types/asana/index.d.ts

```diff
@@ -17,7 +17,6 @@ declare namespace asana {
          * Constructs a Client with instances of all the resources using the dispatcher.
          * It also keeps a reference to the dispatcher so that way the end user can have
          * access to it.
-         * @class
```

Comment:
`@class` is unnecessary when decorating a `new` signature. It's implied by it being `new`.